GOVERNANCE.md

```diff
@@ -165,6 +174,84 @@ We have a reasonably liberal approach to code review and merging. We value quick
 - If a PR is against code you have previously committed and is either small changes, bug fixes, or refactors, then you're free to merge it without any review. However, if you need more confidence in your changes, you can wait for approval from another maintainer or core contributor.
 
+## Financial Contributions
+
+Biome is in the fortunate position that people care enough about it that they want to invest financially in its future. However, with money comes responsibility, so we are committed to the following.
+
+There are two ways to fund Biome and its development: sponsorship and funded bounties, although we strongly prefer sponsorship.
+
+### Sponsorship
+
+Sponsorship is the first and foremost way of making financial contributions to Biome. Sponsorship can be
+done through a one-time donation, or through recurring donations. We offer incentives for certain levels of recurring donations.
+
+To learn more about sponsorship incentives, or to make a donation, please visit our [Open Collective page](https://opencollective.com/biome).
+You can also sponsor the project through [GitHub](https://github.com/sponsors/biomejs).
+
+### Community-Funded Bounties
+
+Bounties are the second approach towards funding Biome development. Unlike a sponsorship, bounties are directed towards the completion of a specific task. Tasks include, but are not limited to, the development of a feature or the fixing of a bug.
+
+Compared to sponsorship, bounties come with a few risks, both for Biome as a project, as well as the persons or entities offering and collecting the bounty. To minimize these risks, we only sanction bounties under the following conditions:
+
+- Bounties must be pledged through Polar.sh on an issue that is opened in our issue tracker.
+  - Unless otherwise specified, Biome reserves 30% of the payment amount, meaning contributors receive 70% of the pledged amount.
+  - The commission charged by Polar.sh is split accordingly.
+- If an issue in our issue tracker is not yet open for pledges, the person who wishes to pledge should indicate their desire in the comments of the issue, or do so through the `Community > #bounties` channel on Discord. A core contributor can then **accept** or **reject** the request for the bounty. Reasons for rejection include, but are not limited to: 
+  - Implementation of the task would conflict with the project direction or [its values](https://biomejs.dev/internals/philosophy/).
+  - Implementation of the task is (currently) technically infeasible.
+  - The suggested pledge is deemed inappropriate for the amount of work involved.
+- The person making the pledge acknowledges their pledge does not come with a guarantee of work done.
+- A task is only completed when a Biome maintainer merges the pull request that closes the task. The person trying to complete the task is responsible for addressing review comments to make sure the work meets the quality standards of the Biome project.
```

Comment:
It was intentional, since maintainers have merge permissions. But if it makes more sense, we can of course include a rule to specify only Core contributors should merge PRs that are for bounties.

Reply:
No, I think it's fine :)